documentation/content/guidebook/github-oauth/$nuxt.md

```diff
@@ -46,9 +46,13 @@ Expose the environment variables by updating your Nuxt config.
 export default defineNuxtConfig({
 	// ...
 	runtimeConfig: {
-		githubClientId: "", // keep it empty!
-		githubClientSecret: "" // keep it empty!
+		githubClientId: process.env.GITHUB_CLIENT_ID,
+		githubClientSecret: process.env.GITHUB_CLIENT_SECRET
```

Comment:
I found that I needed to populate the `githubClientId` and `githubClientSecret` to make auth work.
Let me know if I'm missing something.